api/projects/views.py

```diff
@@ -103,6 +106,20 @@ def user_permissions(self, request: Request, pk: int = None):
         serializer = UserObjectPermissionsSerializer(instance=permission_data)
         return Response(serializer.data)
 
+    @action(detail=True, methods=["POST"], url_path="migrate-to-edge")
```

Comment:
Can we add a swagger auto schema decorator here? I guess with the `EmptySerializer` class?